Source/Csla.Channels.RabbitMq/RabbitMqPortalOptions.cs

```diff
@@ -0,0 +1,45 @@
+﻿//-----------------------------------------------------------------------
+// <copyright file="RabbitMqPortalOptions.cs" company="Marimer LLC">
+//     Copyright (c) Marimer LLC. All rights reserved.
+//     Website: https://cslanet.com
+// </copyright>
+// <summary>Options for RabbitMqProxy</summary>
+//-----------------------------------------------------------------------
+
+namespace Csla.Channels.RabbitMq;
+
+/// <summary>
+/// Options for RabbitMqPortal
+/// </summary>
+public class RabbitMqPortalOptions
+{
+  private Uri? _dataPortalUri;
+  /// <summary>
+  /// Gets or sets the data portal server endpoint URL
+  /// </summary>
+  public Uri? DataPortalUri 
```

Comment:
Document the exceptions thrown here. Probably the easierst way is to remove the current XML doc and let VS autocomplete it again with `///`. It should add the exceptions in the setter. Should save a bit of typing :)